site/docs/tools/iceberg/spark.md

```diff
@@ -4,17 +4,22 @@
     Detailed steps on how to set up Pyspark + Iceberg + Nessie with Python is available on [Binder](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=notebooks/nessie-iceberg-demo-nba.ipynb)
 
 To access Nessie from a spark cluster make sure the `spark.jars` spark option is set to include
-the [Spark 2](https://repo.maven.apache.org/maven2/org/apache/iceberg/iceberg-spark/{{ versions.iceberg }}/iceberg-spark-{{ versions.iceberg }}.jar)
-or [Spark 3](https://repo.maven.apache.org/maven2/org/apache/iceberg/iceberg-spark3/{{ versions.iceberg }}/iceberg-spark3-{{ versions.iceberg }}.jar) or
-[Spark 3.2](https://repo.maven.apache.org/maven2/org/apache/iceberg/iceberg-spark-runtime-{{ versions.iceberg_spark32 }}/{{ versions.iceberg }}/iceberg-spark-runtime-{{ versions.iceberg_spark32 }}-{{ versions.iceberg }}.jar) Nessie plugin jar. This fat jar
-is distributed by the Apache Iceberg project and contains all Apache Iceberg libraries required for operation, including the built-in Nessie Catalog.
+the 
+[Spark 2.4](https://repo.maven.apache.org/maven2/org/apache/iceberg/iceberg-spark-runtime-{{ versions.spark24 }}/{{ versions.iceberg }}/iceberg-spark-runtime-{{ versions.spark24 }}-{{ versions.iceberg }}.jar) or 
+[Spark 3.0](https://repo.maven.apache.org/maven2/org/apache/iceberg/iceberg-spark-runtime-{{ versions.spark30 }}_{{ versions.scala212 }}/{{ versions.iceberg }}/iceberg-spark-runtime-{{ versions.spark30 }}_{{ versions.scala212 }}-{{ versions.iceberg }}.jar) or 
```

Comment:
Please remove the mention of 3.0 + 2.4 (Nessie does not support these Spark versions).